apps/server/src/modules/files-storage/controller/api-test/files-storage-download-upload.api.spec.ts

```diff
@@ -77,39 +78,46 @@ class API {
 
 const createRndInt = (max) => Math.floor(Math.random() * max);
 
+const createFileResponse = (contentRange?: string) => {
+	const text = 'testText';
+	const buffer = Buffer.from(text);
```

Comment:
I think ` const readable = Readable.from(text)` would also do the job and you don't need the other lines.